daprdocs/data/components/state_stores/generic.yaml

```
@@ -42,6 +42,17 @@
     etag: true
     ttl: false
     query: false
+- component: Coherence
+  link: setup-coherence
+  state: Alpha
+  version: v1
+  since: "1.15"
```

This is first in the v1.16 release. You also need to target the v1.16 docs branch. Can you move this PR to that branch?

```suggestion
  since: "1.16"
```